getsolus/packages
# mate-user-admin: Update to 1.7.0

**Summary**
- No summarized changelog provided.
- Make usable from xfce settings, ref #305.

**Test Plan**

- Install on xfce, verify that "Users and Groups" showed up in xfce settings.
- Add and remove users, verify expected default groups.

**Checklist**
